src/tiled/brokenlinks.cpp

```diff
@@ -404,6 +460,7 @@ void BrokenLinksWidget::selectionChanged()
     mLocateButton->setEnabled(!selection.isEmpty());
 
     bool isTileset = qobject_cast<TilesetDocument*>(mBrokenLinksModel->document()) != nullptr;
+    bool isTemplateGroup = qobject_cast<TemplateGroupDocument*>(mBrokenLinksModel->document()) != nullptr;
```

Comment:
The broken links model document can never be a template group document, since we don't currently have a template group editor.